docs/install/windows.markdown

```diff
@@ -9,8 +9,8 @@ You can download a setup.exe style installer to guide the installation
 (requires admin privileges) or a simple zip file and manage the files for
 yourself (no special privileges required).
 
-<a href="{{ windows_exe_stable }}" class="btn">Windows (setup.exe)</a>
-<a href="{{ windows_exe_nightly }}" class="btn">Nightly Windows (setup.exe)</a>
+<a href="{{ windows_exe_stable }}" class="btn">Windows (setup.exe)</a><a href="{{  }}">Setup checksum</a><br/>
```

Comment:
This is how you could refer to one of those new files:

```suggestion
<a href="{{ windows_exe_stable }}" class="btn">Windows (setup.exe)</a><a href="{{ windows_exe_stable }}.sha256">Setup checksum</a><br/>
```

However:
* Very few people ever verify hashes so it doesn't warrant being made into a big button here
* There are no stable releases with hashes yet, so putting that here now would just give a broken link

I think it is more reasonable to have a separate section below that links to the hashes and has instructions for verifying them.

I'm also skeptical that folks will do this on Windows; I don't think they even have the tools installed to this by default.  The other OSs seem more reasonable to target.

Reply:
> I think it is more reasonable to have a separate section below that links to the hashes and has instructions for verifying them.

* yeah, i was thinking exactly same. like some paragraph mentioning smth like: "go to here for getting the sha256 checksum" would be better.

> I'm also skeptical that folks will do this on Windows; 

😂 on point, 

> I don't think they even have the tools installed to this by default.

i guess it does have some CLI tool for that, but 'ts nowhere as lean as GNU coreutils' `sha256sum` or `cksum` (v9.0+)
i am on windows, and doing this before i discovered GNU coreutils via MSYS2 was very involving task - open the tool's GUI, drag n drop files, open the checksum file and copy paste in the tool gui  etc etc. Starting to use GNU coreutils via MSYS2 was the best of the best thing on computer i have learnt/done in years.